packages/webapp/pages/[userId]/index.tsx

```diff
@@ -217,8 +208,23 @@ const ProfilePage = ({ profile }: ProfileLayoutProps): ReactElement => {
       {readingHistory?.userReadingRankHistory && (
         <>
           <ActivityContainer>
-            <ActivitySectionTitle>Weekly goal</ActivitySectionTitle>
-            <div className="flex flex-wrap">
+            <ActivitySectionTitle>
+              <TitleWithLink
+                title="Weekly goal"
+                subtitle="Learn how we count"
+                clickableTitle="weekly goals"
+                link="https://docs.daily.dev/docs/your-profile/weekly-goal"
+              />
+              <Dropdown
+                className="hidden laptop:block ml-auto"
+                selectedIndex={selectedHistoryYear}
+                options={dropdownOptions}
+                onChange={(val, index) => setSelectedHistoryYear(index)}
+                buttonSize="small"
+                style={{ width: '8rem', minWidth: 'fit-content' }}
```

Comment:
Can't we use classnames for this?

Reply:
I think we can. This was just copy and pasted from the previous implementation - forgot to update. Nice catch!